zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/usermanagement/AuthorizationController.java

```diff
@@ -40,8 +40,9 @@ public CompletableFuture<ResponseEntity<Object>> patchAuthorization(
       @PathVariable final long ownerKey,
       @RequestBody final AuthorizationPatchRequest authorizationPatchRequest) {
 
-    return RequestMapper.toAuthorizationPatchRequest(ownerKey, authorizationPatchRequest)
-        .fold(RestErrorMapper::mapProblemToCompletedResponse, this::patchAuthorization);
+    //    return RequestMapper.toAuthorizationPatchRequest(ownerKey, authorizationPatchRequest)
+    //        .fold(RestErrorMapper::mapProblemToCompletedResponse, this::patchAuthorization);
+    throw new NotImplementedException("Authorization patching is not yet implemented");
```

Comment:
❓ Is this expected?

Reply:
Yes, https://github.com/camunda/camunda/pull/22775